index.d.ts

```diff
@@ -295,6 +300,17 @@ declare namespace execa {
 		Promise<ExecaReturnValue<StdoutErrorType>>;
 }
 
+interface killOptions {
+	/**
+	If the first signal does not terminate the child after a specified timout, a `SIGKILL` will be sent to the process.
+	*/
+	retry: boolean;
+	/**
```

Comment:
```suggestion

	/**
```

Reply:
I think this one disappeared through other changes :o